CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+* Add `uiSchema` to the Footer component.
```

Comment:
The keep changelog pattern is with '-' instead of the '*'

https://keepachangelog.com/en/1.0.0/

Reply:
Top